src/fr/hentaiorigines/src/eu/kanade/tachiyomi/extension/fr/hentaiorigines/HentaiOrigines.kt

```diff
@@ -0,0 +1,25 @@
+package eu.kanade.tachiyomi.extension.fr.hentaiorigines
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import eu.kanade.tachiyomi.network.POST
+import okhttp3.Request
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class HentaiOrigines : Madara(
+    "Hentai Origines",
+    "https://hentai-origines.fr",
+    "fr",
+    dateFormat = SimpleDateFormat("MMMM dd, yyyy", Locale("fr")),
+) {
+    override val mangaDetailsSelectorAuthor = "div.manga-authors a"
+    override val mangaDetailsSelectorDescription = "div.summary__content"
+    override val seriesTypeSelector = ".post-title span"
+
+    override val useLoadMoreRequest = LoadMoreStrategy.Always
+    override val useNewChapterEndpoint = true
+
+    override fun xhrChaptersRequest(mangaUrl: String): Request {
+        return POST("$mangaUrl/ajax/chapters/", xhrHeaders)
```

Comment:
what is the difference here?

Reply:
`$mangaUrl/ajax/chapters` returns a 4xx error, `$mangaUrl/ajax/chapters/` works

Reply:
ah the trailing slash